javascript-carousel/main.js

```diff
@@ -0,0 +1,129 @@
+var $image = document.querySelector('.carousel-image');
+var $left = document.querySelector('.left');
+var $right = document.querySelector('.right');
+var $round = document.querySelectorAll('.round');
+
+var image = 0;
+
+function removeDot(number) {
+  for (var i = 0; i < $round.length; i++) {
+    if ($round[i] === $round[number]) {
+      $round[i].className = 'fa-solid fa-circle round';
+    } else {
+      $round[i].className = 'fa-regular fa-circle round';
+    }
+  }
+}
+
+function timeReset() {
+  clearInterval(interval);
+  interval = setInterval(slideImage, 3000);
+}
+
+function slideImage() {
+  image++;
+  if (image === 1) {
+    $image.setAttribute('src', 'images/004.png');
+    removeDot(1);
+  } else if (image === 2) {
+    $image.setAttribute('src', 'images/007.png');
+    removeDot(2);
+  } else if (image === 3) {
+    $image.setAttribute('src', 'images/025.png');
+    removeDot(3);
+  } else if (image === 4) {
+    $image.setAttribute('src', 'images/039.png');
+    removeDot(4);
+  } else if (image === 5) {
+    $image.setAttribute('src', 'images/001.png');
+    image = 0;
+    removeDot(0);
+  }
+}
+
+function lastPicture(event) {
+  timeReset();
+  image--;
+  if (image === 0) {
+    $image.setAttribute('src', 'images/001.png');
+    removeDot(0);
+  } else if (image === 1) {
+    $image.setAttribute('src', 'images/004.png');
+    removeDot(1);
+  } else if (image === 2) {
+    $image.setAttribute('src', 'images/007.png');
+    removeDot(2);
+  } else if (image === 3) {
+    $image.setAttribute('src', 'images/025.png');
+    removeDot(3);
+  } else if (image === 4) {
+    $image.setAttribute('src', 'images/039.png');
+    removeDot(4);
+  } else if (image < 0) {
+    image = 4;
+    $image.setAttribute('src', 'images/039.png');
+    removeDot(4);
+  }
+}
+
+function nextPicture(event) {
+  timeReset();
+  image++;
+  if (image === 1) {
+    $image.setAttribute('src', 'images/004.png');
+    removeDot(1);
+  } else if (image === 2) {
+    $image.setAttribute('src', 'images/007.png');
+    removeDot(2);
+  } else if (image === 3) {
+    $image.setAttribute('src', 'images/025.png');
+    removeDot(3);
+  } else if (image === 4) {
+    $image.setAttribute('src', 'images/039.png');
+    removeDot(4);
+  } else if (image === 5) {
+    $image.setAttribute('src', 'images/001.png');
+    image = 0;
+    removeDot(0);
+
+  }
```

Comment:
You're using the `image` variable here a lot like indexes of an array. If you make the list of image URLs into an array, you might be able to simplify this code by quite a lot.